src/index.tsx

```diff
@@ -1,2 +1,58 @@
-import React from 'react'
+import React from "react";
+import { Form } from "./components/form";
+import { ModalContainer } from "./components/modal-container";
+import { MoodSelector } from "./components/mood-selector";
+import { Mood, MOODS_DATA, MoodList } from "./moods";
+import { styled } from "./theme";
 
+export const DEFAULT_MOODS_LIST = MOODS_DATA["msFluent"];
+
+export interface FeedbackData {
+  mood: {
+    value: number;
+    total: number;
+    label: string;
+    id: string;
+  };
+  comment: string;
+}
+
+const Container = styled("div", {
+  display: "flex",
+  flexDirection: "column",
+  alignItems: "center",
+  maxWidth: "360px",
+  position: "relative",
+});
+
+type HandleFeedbackSubmit = (data: FeedbackData) => void;
+
+interface CheekyFeebackProps {
+  handleSubmit: HandleFeedbackSubmit;
+  moods: MoodList;
+  /**
+   * Text to be displayed in the header of the form.
+   */
```

Comment:
lets not add this comment right now. We can think about how to convey the message properly later on